src/psyclone/tests/domain/nemo/transformations/nemo_arrayrange2loop_trans_test.py

```diff
@@ -71,48 +71,9 @@ def test_transform():
 
 
 def test_apply_bounds(tmpdir):
-    '''Check that the apply method uses a) configuration bounds if they
-    are provided or b) lbound and ubound intrinsics when no bounds
-    information is available. Also check that a NemoKern is added
-    between the assignment and the innermost enclosing loop after the
-    last range has been transformed into an explicit loop.
-
-    '''
-    _, invoke_info = get_invoke("implicit_many_dims.f90", api=API, idx=0)
-    schedule = invoke_info.schedule
-    assignment = schedule[0]
-    array_ref = assignment.lhs
-    trans = NemoArrayRange2LoopTrans()
-    for index in range(4, -1, -1):
-        assert not schedule.walk(NemoKern)
-        range_node = array_ref.children[index]
-        trans.apply(range_node)
-    assert schedule.walk(NemoKern)
-    assert isinstance(assignment.parent, Schedule)
-    assert isinstance(assignment.parent.parent, NemoKern)
-    assert isinstance(assignment.parent.parent.parent, Schedule)
-    assert isinstance(assignment.parent.parent.parent.parent, NemoLoop)
-    assert assignment.parent.parent.parent.parent.loop_type == "lon"
-    writer = FortranWriter()
-    result = writer(schedule)
-    assert (
-        "  do idx = LBOUND(umask, 5), UBOUND(umask, 5), 1\n"
-        "    do jt = 1, UBOUND(umask, 4), 1\n"
-        "      do jk = 1, jpk, 1\n"
-        "        do jj = 1, jpj, 1\n"
-        "          do ji = 1, jpi, 1\n"
-        "            umask(ji,jj,jk,jt,idx) = vmask(ji,jj,jk,jt,idx) + 1.0\n"
-        "          enddo\n"
-        "        enddo\n"
-        "      enddo\n"
-        "    enddo\n"
-        "  enddo" in result)
-    assert Compile(tmpdir).string_compiles(result)
-
-
-def test_apply_fixed_bounds(tmpdir):
```

Comment:
We still need this test and associated functionality.

Reply:
Again, I did merge both tests because I don't think this is a special case for this transformation, but note that the middle loop in the remaining test has fixed ranges and the generated code does respect them.